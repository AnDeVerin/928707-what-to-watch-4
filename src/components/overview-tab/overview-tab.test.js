import React from 'react';
import renderer from 'react-test-renderer';

import OverviewTab from './overview-tab.jsx';

const overview = {
  rating: {
    value: 6.6,
    count: 240,
  },
  description: `At the end of the first film, the powerful Dark wizard Gellert Grindelwald (Depp) was captured by MACUSA (Magical Congress of the United States of America), with the help of Newt Scamander (Redmayne). But, making good on his threat, Grindelwald escaped custody and has set about gathering followers, most unsuspecting of his true agenda: to raise pure-blood wizards up to rule over all non-magical beings. In an effort to thwart Grindelwald's plans, Albus Dumbledore (Law) enlists his former student Newt Scamander, who agrees to help, unaware of the dangers that lie ahead. Lines are drawn as love and loyalty are tested, even among the truest friends and family, in an increasingly divided wizarding world.`,

  director: `David Yates`,
  stars: [
    `Johnny Depp`,
    `Eddie Redmayne`,
    `Kevin Guthrie`,
    `Carmen Ejogo`,
    `Wolf Roth`,
    `Eddie Redmayne`,
    `ZoÃ« Kravitz`,
    `Callum Turner`,
    `Derek Riddell`,
    `Cornell John`,
    `Ezra Miller`,
  ],
  runTime: 111,
};

it(`OverviewTab component renders correctly`, () => {
  const tree = renderer.create(<OverviewTab overview={overview} />).toJSON();

  expect(tree).toMatchSnapshot();
});
