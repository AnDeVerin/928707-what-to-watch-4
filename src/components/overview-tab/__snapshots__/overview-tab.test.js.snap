// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OverviewTab component renders correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      6.6
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Good
      </span>
      <span
        className="movie-rating__count"
      >
        240
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      At the end of the first film, the powerful Dark wizard Gellert Grindelwald (Depp) was captured by MACUSA (Magical Congress of the United States of America), with the help of Newt Scamander (Redmayne). But, making good on his threat, Grindelwald escaped custody and has set about gathering followers, most unsuspecting of his true agenda: to raise pure-blood wizards up to rule over all non-magical beings. In an effort to thwart Grindelwald's plans, Albus Dumbledore (Law) enlists his former student Newt Scamander, who agrees to help, unaware of the dangers that lie ahead. Lines are drawn as love and loyalty are tested, even among the truest friends and family, in an increasingly divided wizarding world.
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        David Yates
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Johnny Depp, Eddie Redmayne, Kevin Guthrie
         and other
      </strong>
    </p>
  </div>,
]
`;
